src/globals/js/watch.js

```diff
@@ -0,0 +1,66 @@
+import initCheckbox from '../../components/checkbox/checkbox';
+import { componentClasses } from '../../index';
+
+const forEach = Array.prototype.forEach;
```

Comment:
@chrisdhanaraj Good question - `[…NodeList].forEach()` is convenient, but it creates array copy of node list, causing memory pressure. Given the mutation observer callback runs upon every DOM node addition/removal, which means we’d expect more frequent calls there than regular component code, I avoided any coding approach that hits performance.